internal/notify/notifications.lua

```diff
@@ -303,8 +303,35 @@ local function get_bar(get_fn, get_max_fn, text, color)
     return nil
 end
 
+local function getSaveAlert()
```

Comment:
```suggestion
local function get_save_alert()
```

Reply:
make sure you follow the style of the surrounding code. variables below should also be in snake_case.